docs/docs.css

```diff
@@ -102,6 +102,8 @@ summary {
   display: inline;
 }
 
+summary::-webkit-details-marker { display: none; }
```

Comment:
```suggestion
summary::-webkit-details-marker {
    display: none;
}
```
The project uses this kind of styling for single entries.

Reply:
I'm not picky :) This is fine!